package.json

```diff
@@ -17,18 +17,6 @@
     "build:dist": "babel index.js --source-root ./ -d ./dist",
     "build:docs": "documentation build ./index.js --github --sort-order='alpha' -f md > ./docs/index.md"
   },
-  "browserify": {
```

Comment:
maybe leave this as a script for containers etc are not forced to have a global browserify in PATH and only `npm ...` scripts can access the `.bin` dir?

Reply:
I think this was never meant to be used for ``browserify`` itself but is just used internally for ``karma`` testing. Or am I wrong here?

Reply:
Yes there is no browserify in the build for this module - it's only for karma